workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx

```diff
@@ -143,7 +214,45 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     flexShrink: 0,
     flexGrow: 0,
+    flexDirection: 'row-reverse',
+    marginLeft: theme.spacing(1),
+
+    '& > *:nth-child(n + 2)': {
+      // all but the first child
+      marginRight: theme.spacing(1),
+    },
+  },
+  commitCompareButton: {
+    ...button(),
+  },
+  commitDocsButton: {
+    ...button(),
+
+    background: '#fff',
+  },
+
+  commitResetButton: {
+    ...button('outlined'),
+  },
+
+  commitControlIcon: {
+    width: theme.spacing(2),
+    marginRight: theme.spacing(0.5),
   },
-  commitCompareButton: {},
-  commitResetButton: {},
 }));
+
+function button(variant?: 'outlined'): CSSProperties {
+  return {
+    fontFamily: FontFamily,
+    color: OpticBlueReadable,
+    textTransform: 'none',
+
+    ...(variant === 'outlined'
+      ? {
+          border: `1px solid ${LightBlueBackground}`,
+        }
+      : {
+          background: LightBlueBackground,
+        }),
+  };
+}
```

Comment:
This is a very quick and dirty version of a mixin that could eventually live on the theme object. Over time, those should be the source of truth of what buttons look like in Optic, with generic a `Button` component applying the default styles. That way, defaults can be used, but customisation remains straightforward as well.

However, that's way beyond the scope of what we can justify to do here, so this will just have to be the start.